platforms/Windows/shared/shared.wxs

```diff
@@ -1,7 +1,15 @@
 ﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
   <Fragment>
-    <StandardDirectory Id="ProgramFiles6432Folder">
-      <Directory Id="INSTALLDIR" Name="Swift" />
+    <StandardDirectory Id="LocalAppDataFolder">
+      <Directory Name="Programs">
+        <Directory Id="INSTALLROOT" Name="Swift">
+          <Directory Id="INSTALLDIR" Name="$(MajorMinorProductVersion)" />
```

Comment:
Oh wait, this is swapping the order of the versioned component and the component?  Can we restore the original order?  e.g.

```
Swift
  Redistributables
     5.8.1
     5.8.2
  Runtimes
     5.8.1
     5.8.2
  Toolchains
     5.8.1+Asserts
     5.8.2+NoAsserts
```

Reply:
Depends on what you mean by "original" -- the current root is the version (the change was to swap it for Redistributables). That's a little easier because there's only one versioned directory. But there are only five directories so it's not a big deal. Do you want me to make that change? If so, should it be three-field version or just major.minor? (Remember that 5.8.2 would remove 5.8.1.)

Reply:
Good question ... so I do think that there is some value in the full 3 component field - it makes it easy to identify the version installed.  I'm thinking about sinking the version number because for the toolchains, we may want to have 2 parallel installations of the same version (`+Asserts` and `+NoAsserts` variants).  Having that level of flexibility for the other component seems useful as well (e.g. possibly enabling additional debug variants some day on the runtimes).

Reply:
OK, will do.